templates/500.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base/base.html" %}
+{% load static %}
+
+{% block content %}
+
+  <div class="govuk-width-container">
+    <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-l">Server error</h1>
+          <p class="govuk-body">
+            There is a problem with the server.
```

Comment:
Maybe we could link to our slack channel as well - like if the problem persists let us know here